tests/sub/test_sub.py

```diff
@@ -0,0 +1,72 @@
+# general imports
```

Comment:
This is a new test that makes sure that subtraction of large integers works in **msdsl**.  I verified that this test failed when run using the previous version of **msdsl**.